### PR TITLE
Added manual preview script

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Allows users to kick off previews locally if Travis CI Netlify job fails to run. Include the following changes:
- Added a new `scripts/manualpreview.sh` script. Users can use it to send a manual request to kick of the preview build. 

Both `autopreview` and `manualpreview` scripts uses different endpoints on purpose, in case if auto preview stops working, users can at least use the manual one.

Cherry-pick into:
- `enterprise-4.13`
- `enterprise-4.12`
- `enterprise-4.11`
- `rhacs-docs`
- `rhacs-docs-4.1`
- `rhacs-docs-4.0`

Thank you @amolnar-rh I took most of the code from your script.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@aireilly @amolnar-rh Can you please review this PR?

File: scripts/manualpreview.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+hash gh 2>/dev/null || { echo >&2 "Error: GitHub CLI (gh) is not installed. Please install gh before running this script."; exit 1; }
+
+# Cehck if inside a repository
+if ! git rev-parse --is-inside-work-tree > /dev/null 2>&1; then
+  echo "ERROR: you must execute this script from the OpenShift Git repository." 1>&2
+  exit 1
+fi
+
+PR_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+PR_NUMBER="$(gh pr list -H $PR_BRANCH --json number | jq -r '.number')"
```
Comment (aireilly) [inline scripts/manualpreview.sh:18]: https://www.shellcheck.net/wiki/SC2086

```suggestion
PR_NUMBER="$(gh pr list -H "$PR_BRANCH" --json number | jq -r '.number')"
```

Comment (aireilly) [reply scripts/manualpreview.sh:18]: bash-lsp tells me to do this, but I can't get the script to run so :shrug:

**Review by aireilly — COMMENTED** [review]
Script does not run for me. 

```cmd
$ ./scripts/manualpreview.sh 
jq: error (at <stdin>:1): Cannot index array with string "number"

```

File: scripts/manualpreview.sh (hunk 1 above)
Comment (amolnar-rh) [inline scripts/manualpreview.sh:18]:
```suggestion
PR_NUMBER="$(gh pr list -H $PR_BRANCH --json number | jq -r '.[0].number')"
```

@gaurav-nelson You might have used an earlier version of the script, I remember I had a problem with jq's syntax here. This should fix the `[jq -r '.number](jq: error (at <stdin>:1)` issue :)

**Comment by aireilly** [comment]
tested locally, works well and runs fast.

/lgtm

**Review by aireilly — APPROVED** [review]

File: scripts/manualpreview.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+set -e
+
+# Check if jq is installed
+hash jq 2>/dev/null || { echo >&2 "Error: jq is not installed. Please install jq before running this script."; exit 1; }
+
+# Check if gh is installed
+hash gh 2>/dev/null || { echo >&2 "Error: GitHub CLI (gh) is not installed. Please install gh before running this script."; exit 1; }
+
+# Cehck if inside a repository
```
Comment (aireilly) [inline scripts/manualpreview.sh:11]:
```suggestion
# Check if inside a repository
```

**Comment by bergerhoffer** [comment]
@gaurav-nelson Can you cherry pick this to 4.14 too please, since it targets main?

**Pull request merged**